brainstormforce/ultimate-addons-for-gutenberg
# Fix: Forms and Icon-List - Throws an error on drag and drop.

### Description

- Run time changes.

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a new page.
- Drag and drop Forms block and Icon List from Block Inserter.
- For Forms - Check whether it is rendering properly or not.
- Try switching to responsive and check the console.
- For Icon List - There should be no console when dragging and dropping.

### Checklist:
